DEPLOYMENTS_MODULAR.md

```diff
@@ -0,0 +1,19 @@
+# Contract Deployments
+
+## Mainnets
+
+| Contract Name | Network | Contract Address | Transaction Hash |  
+| --- | --- | --- |  --- |
+| `ModularEtherspotWallet` |   | []() | []() |
+| `ModularEtherspotWalletFactory` |   | []() | []() |
+| `Bootstrap` |   | []() | []() |
+| `MultipleOwnerECDSAValidator` |   | []() | []() |
+
+## Testnets
+
+| Contract Name | Network | Contract Address | Transaction Hash |  
+| --- | --- | --- |  --- |
+| `ModularEtherspotWallet` | `sepolia` | [0x917398df969752a7d94725740918f38e4b8eecec](https://sepolia.etherscan.io/address/0x917398df969752a7d94725740918f38e4b8eecec) | [0x983a9141f38e7e1a1a70cfe9354eb5e6d61dd39e228990b80146c175a1928d6e](https://sepolia.etherscan.io/tx/0x983a9141f38e7e1a1a70cfe9354eb5e6d61dd39e228990b80146c175a1928d6e) |
+| `ModularEtherspotWalletFactory` | `sepolia` | [0x77e4288a4b15893f520f15c262a07df9866904e4](https://sepolia.etherscan.io/address/0x77e4288a4b15893f520f15c262a07df9866904e4) | [0x0f4b4f53a2654e34e7b0a3aad01f9a90d711df2af1d3d9d8a2e70a63c36bc254](https://sepolia.etherscan.io/tx/0x0f4b4f53a2654e34e7b0a3aad01f9a90d711df2af1d3d9d8a2e70a63c36bc254) |
+| `Bootstrap` | `sepolia` | [0x4f695ad7694863c8280fcebf2cb220e361ce4ea0](https://sepolia.etherscan.io/address/0x4f695ad7694863c8280fcebf2cb220e361ce4ea0) | [0xc69aed3fd25ef9bd66c42903dd855cd363d703f03049be9fb3a2fdf0c75ecfb1](https://sepolia.etherscan.io/tx/0xc69aed3fd25ef9bd66c42903dd855cd363d703f03049be9fb3a2fdf0c75ecfb1) |
+| `MultipleOwnerECDSAValidator` | `sepolia` | [0x1e714c551fe6234b6ee406899ec3be9234ad2124](https://sepolia.etherscan.io/address/0x1e714c551fe6234b6ee406899ec3be9234ad2124) | [0x1b56d1e32cb12d16b3878eed660ae23eb454cd3e84ae9e88d8ed40d0d0c5e6f8](https://sepolia.etherscan.io/tx/0x1b56d1e32cb12d16b3878eed660ae23eb454cd3e84ae9e88d8ed40d0d0c5e6f8) |
```

Comment:
Can we pls update README to demonstrate how SessionKeys can be used? Also, do we have benchmarks on the gas associated with session keys deployment.

Reply:
done